dive_core/command_hierarchy.cpp

```diff
@@ -1622,35 +1568,27 @@ std::string CommandHierarchyCreator::GetEventString(const IMemoryManager &mem_ma
     {
         PM4_CP_DRAW_INDX packet;
         DIVE_VERIFY(mem_manager.CopyMemory(&packet, submit_index, va_addr, sizeof(packet)))
-        string_stream << "DrawIndexOffset(NumIndices:" << packet.bitfields2.NUM_INDICES << ","
-                      << "IndexBase:" << packet.bitfields3.INDX_BASE << ","
-                      << "IndexSize:" << packet.bitfields4.INDX_SIZE << ","
-                      << "VizQuery:" << packet.bitfields0.VIZ_QUERY << ")";
+        string_stream << "DrawIndexOffset(NumIndices:" << packet.bitfields2.NUM_INDICES << ")";
```

Comment:
I removed the primtype, index base, index size and viz query from both ui and the lrz cli, found them not quite useful, we can add them back if needed

Reply:
Sounds good! The user can get them from the PM4 packet anyways.